alfasim_sdk/variables.py

```diff
@@ -3,19 +3,19 @@
 from typing import Optional
 
 import attr
+from alfasim_sdk._validators import check_string_is_not_empty, check_unit_is_valid
```

Comment:
This import seems strange, should be kept alone...

Reply:
Another idea is that we can try using https://github.com/asottile/reorder_python_imports instead, which should present fewer problems and has native pre-commit integration. 👍

Reply:
Sorry, I saw your comments just after adding the `seed-isort-config` on pre-commit, the next commit will add your suggestion ;)